cookbook/bundles/best_practices.rst

```diff
@@ -117,20 +117,20 @@ files are going to be part of the repository.
 
 The following classes and files have specific emplacements:
 
-===============================  =============================
-Type                             Directory
-===============================  =============================
-Commands                         ``Command/``
-Controllers                      ``Controller/``
-Service Container Extensions     ``DependencyInjection/``
-Event Listeners                  ``EventListener/``
-Model classes [1]                ``Model/``
-Configuration                    ``Resources/config/``
-Web Resources (CSS, JS, images)  ``Resources/public/``
-Translation files                ``Resources/translations/``
-Templates                        ``Resources/views/``
-Unit and Functional Tests        ``Tests/``
-===============================  =============================
+===============================  =============================  ================
+Type                             Directory                      Location
+===============================  =============================  ================
+Commands                         ``Command/``                   Mandatory
+Controllers                      ``Controller/``                Mandatory
+Service Container Extensions     ``DependencyInjection/``       Mandatory
+Event Listeners                  ``EventListener/``             Convention
+Model classes [1]                ``Model/``                     Convention
+Configuration                    ``Resources/config/``          Mandatory
+Web Resources (CSS, JS, images)  ``Resources/public/``          Mandatory
+Translation files                ``Resources/translations/``    Mandatory
+Templates                        ``Resources/views/``           Mandatory
+Unit and Functional Tests        ``Tests/``                     Mandatory
```

Comment:
this is not really mandatory. It is perfectly fine to put them elsewhere (even outside the prod code of your bundle) as long as you configure your testing tool accordingly (and it is perfectly fine to use another tool than PHPUnit, in which case tests would be elsewhere)